scripts/spidhub-fargate/environments/beta/params.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Parameters": {
+    "Project": "spidhub",
+    "Environment": "beta",
+    "Storage": "spidhub-beta-eu-west-1",
```

Comment:
Non so se può valere la pena fare la sostituzione dei parametri nel _setup.sh_ con
`jq ".Parameters += {\"Environment\": \"$ENVIRONMENT\", \"Storage\": \"$PROJECT-$ENVIRONMENT-$AWS_REGION\" }" params.json`

Reply:
In ogni caso quel file andrebbe modificato con altri valori specifici... forse non lo farei